ush/setup.sh

```diff
@@ -1211,6 +1211,7 @@ if [ "${RUN_ENVIR}" = "nco" ]; then
   check_for_preexist_dir_file "${CYCLE_BASEDIR}" "${PREEXISTING_DIR_METHOD}"
   COMROOT="$PTMP/com"
   COMOUT_BASEDIR="$COMROOT/$NET/$envir"
+  NEGES_BASEDIR="$NWGES/$envir/$NET"
```

Comment:
COMOUT_BASEDIR="$COMROOT/$NET/$envir"
  NEGES_BASEDIR="$NWGES/$envir/$NET"

Do you know why `$NET `and `$envir ` switch positions in COMOUT and NWGES?

Also, should NEGES_BASEDIR be "NWGES_BASEDIR"?

Reply:
Changed to NWGES.

I setup NWGES follow operation.
But I did not check how COMOUT based should be. Will try to check it when get on WCOSS.